internal/runners/initialize/init.go

```diff
@@ -184,3 +181,18 @@ func run(params *RunParams, out output.Outputer) (string, error) {
 
 	return params.Path, nil
 }
+
+func deriveVersion(lang language.Language, version string) string {
+	if version == "" {
+		version = lang.RecommendedVersion()
+		langs, err := model.FetchSupportedLanguages(model.HostPlatform)
+		if err == nil {
```

Comment:
Should at least log the error here. And imo too much logic inside this conditional, rather invert it as well.